src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -557,8 +557,8 @@ def _add_role_assignment(cmd, role, service_principal_msi_id, is_service_princip
             if ex.message == 'The role assignment already exists.':
                 break
             logger.info(ex.message)
-        except:  # pylint: disable=bare-except
-            pass
+        except Exception as ex:  # pylint: disable=broad-except
+            logger.warning(str(ex))
```

Comment:
Similarly, with the following changes, we can distinguish between cli errors and python errors by log level (warning/error).

```suggestion
        except CLIError as ex:
            logger.warning(str(ex))
        except Exception as ex:  # pylint: disable=broad-except
            logger.error(str(ex))
```